torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -722,14 +806,10 @@ def group_batch_fusion_passes(graph: torch.fx.Graph, pre_grad=True):
     print_graph(graph, "Before group_batch fusion in post grads pass.")
     fusions: List[GroupBatchFusionBase] = []
 
-    if pre_grad:
-        fusions = generate_fusion_from_config(
-            config.pre_grad_fusion_options, pre_grad=True
-        )
-    elif has_fbgemm:  # Only group fusion (which needs fbgemm) in post grad.
-        fusions = generate_fusion_from_config(
-            config.post_grad_fusion_options, pre_grad=False
-        )
+    fusion_options = (
+        config.pre_grad_fusion_options if pre_grad else config.post_grad_fusion_options
+    )
+    fusions = generate_fusion_from_config(fusion_options, pre_grad=pre_grad)
```

Comment:
We need to check ```has_fbgemm```, as we can't run group fusion if fbgemm is not available. I'd like to add a ```need_fbgemm``` attribute to ```GroupBatchFusionBase``` and enable the fusion pass if the check passed.

Reply:
Right, I will fix this on top of https://github.com/pytorch/pytorch/pull/114778

Reply:
Hi @yanboliang , I have addressed your comment, can you help take another look?